src/main/java/uk/gov/hmcts/divorce/notification/EmailTemplateName.java

```diff
@@ -55,6 +55,7 @@ public enum EmailTemplateName {
     SOLE_RESPONDENT_AOS_SUBMITTED,
     SOLE_APPLICANT_DISPUTED_AOS_SUBMITTED,
     SOLE_RESPONDENT_DISPUTED_AOS_SUBMITTED,
+    SOLE_AOS_SUBMITTED_SOLICITOR,
```

Comment:
I think it's better to specify if the solicitor here is for applicant1 or applicant2